src/App.jsx

```diff
@@ -14,18 +22,39 @@ function App() {
   const [color, setColor] = useState("#000");
   const [bgColor, setBgColor] = useState("#B7BABF");
   const [showMenuAndBgColor, setShowMenuAndBgColor] = useState(true);
-
-  const BUY_ME_COFFEE_LINK = "https://buymeacoffee.com/mastermickey"
-
+  const [modal,setModal]=useState(false)
+  const style={
+    guideline:`p-4 flex text-xs`,
+  }
   useEffect(() => {
     const canvas = canvasRef.current;
     if (canvas) {
       startDrawing(canvas, color, thickness, bgColor);
     }
   }, [bgColor, color, thickness]);
-
+  const showGuidelines=()=>{
+    setModal(!modal)
+  }
+  const closeModal=()=>{
+    setModal(false)
+  }
   return (
     <>
+        <div className="relative">
+          <div className="flex flex-col justify-center text-center items-center bg-gray-800 pb-8 pt-8">
+          <h1 className="headingfont text-4xl font-bold text-slate-100 p-2">Draw it Out!</h1>
+          <p className="text-gray-500 text-xs">All you need is a canvas to craft your ideas.</p>
+          </div>
+          <div className="absolute top-0 right-0 p-6">
+          <FaBookOpen 
+          size={28} 
+          color="white" 
+          className="bg-black p-1 rounded-xl"
+          onClick={showGuidelines}
+          />
+          </div>
```

Comment:
Could you please change wrapper `div` element into `button` ?

Also you need to add `aria-label` with value of `"show-guidelines"`.

This will helps us to make the button accessible.

Reply:
@GayathriThejus, You need to resolve the conflicts also. Please resolve them ASAP otherwise they will increase on each PR Merge.

Reply:
Have made the necessary changes & resolved conflicts. @0xabdulkhalid Kindly check if issue still persists